pkg/dogstatsd/replay/capture.go

```diff
@@ -22,8 +23,11 @@ type TrafficCapture struct {
 
 // NewTrafficCapture creates a TrafficCapture instance.
 func NewTrafficCapture() (*TrafficCapture, error) {
-	location := config.Datadog.GetString("dogstatsd_uds_capture_path")
-	writer := NewTrafficCaptureWriter(location, config.Datadog.GetInt("dogstatsd_uds_capture_depth"))
+	location := config.Datadog.GetString("dogstatsd_capture_path")
+	if location == "" {
+		location = path.Join(config.Datadog.GetString("run_path"), "dsd_capture")
+	}
+	writer := NewTrafficCaptureWriter(location, config.Datadog.GetInt("dogstatsd_capture_depth"))
```

Comment:
Unless I'm missing something `dogstatsd_capture_depth` also needs to be added to `config.go` with a sensible default.

Reply:
I'll make it explicit, the default value is currently 0 - blocking reads off the channel. This was fine because `config.Datadog.GetInt("dogstatsd_capture_depth")` will return the zero value for int, so `0` and all is good.

Reply:
Thanks, FYI it's also important to mention all options in `config.go` (even if it's just with `SetKnown`) otherwise if the config package doesn't know about some options and will print warnings if it finds them in the config file (so that options that actually don't exist are made visible to the user)